### PR TITLE
Update: Update to Ember 3.13.1

### PR DESCRIPTION
**Pull request by jkusa**
Resolves #

## Description

Upgrade to Ember 3.13.1

## Proposed Changes

- Ember 3.13.1
- Remove jQuery
- Remove mixins

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/data/.travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/data/.travis.yml]: Also added this file to the gitignore file

File: packages/data/addon/services/keg.js [hunk 2]
```diff
@@ -45,15 +44,10 @@ export default Service.extend({
...
   resetByType(type) {
-    let recordKegs = get(this, 'recordKegs'),
-      idIndexes = get(this, 'idIndexes');
-
+    const { recordKegs, idIndexes } = this;
     idIndexes[type] = {};
     recordKegs[type] = A();
-
-    set(this, 'recordKegs', recordKegs);
-    set(this, 'idIndexes', idIndexes);
```
[the remark this answers is not in the document]
Comment (jkusa) [reply packages/data/addon/services/keg.js]: I don't think so. We also use this for testing:

https://github.com/yahoo/navi/search?utf8=✓&q=resetByType&type=

File: packages/data/addon/services/keg.js [hunk 3]
```diff
@@ -80,30 +74,30 @@ export default Service.extend({
...
-
-    let returnedRecords = A();
+    const factory = this._getFactoryForType(options.modelFactory || type);
+    const recordKeg = this._getRecordKegForType(type);
+    const idIndex = this._getIdIndexForType(type);
+    const identifierField = factory.identifierField || KegService.identifierField;
+    const owner = getOwner(this);
 
+    const returnedRecords = A();
     for (let i = 0; i < rawRecords.length; i++) {
```
Comment (mattgraham1995) [inline packages/data/addon/services/keg.js]: Could we turn this into a map function?

Comment (jkusa) [reply packages/data/addon/services/keg.js]: Since the record length can be large, `for` loops are faster: https://jsperf.com/for-loop-vs-map-vs-foreach

**Pull request merged**